config/composer/typo3-104-composer-packages-extensions.php

```diff
@@ -0,0 +1,1089 @@
+    $containerConfigurator->import(__DIR__ . '/../services.php');
+    $services = $containerConfigurator->services();
+    $composerExtensions = [
+        new PackageAndVersion('7elix/styleguide', '^10.0.6'),
+        new PackageAndVersion('aaw-team/pagenotfoundhandling', '^3.0.2'),
+        new PackageAndVersion('agrosup-dijon/bulma-package', '^1.1.1'),
+        new PackageAndVersion('aimeos/aimeos-typo3', '^20.01.1'),
+        new PackageAndVersion('aimeos/aimeos-typo3-dist', '^20.10.0'),
+        new PackageAndVersion('aimeos/aimeos_dist', '^20.10.0'),
+        new PackageAndVersion('alexander-nitsche/typo3-nginx-connector', '^1.0.2'),
+        new PackageAndVersion('amartinno1/ama-t3-upgrade-assistant', '^2.0.0'),
+        new PackageAndVersion('amazing/gh-randomcontent', '^0.9.2'),
+        new PackageAndVersion('amazing/media2click', '^0.2.5'),
+        new PackageAndVersion('ameos/ameos_dailymotion', '^1.0.7'),
+        new PackageAndVersion('ameos/ameos_form', '^1.4.7'),
+        new PackageAndVersion('ameos/ameos_mailredirect', '^1.2.0'),
+        new PackageAndVersion('andersundsehr/aus-driver-amazon-s3', '^1.3.2'),
+        new PackageAndVersion('andreaskastl/calendarize-address', '^1.2.0'),
+        new PackageAndVersion('andreaskastl/openweatherapi', '^3.1.0'),
+        new PackageAndVersion('aoe/addthis', '^1.1.2'),
+        new PackageAndVersion('aoe/aoe-dbsequenzer', '^1.0.0'),
+        new PackageAndVersion('aoe/aoe_ipauth', '^3.0.0'),
+        new PackageAndVersion('aoe/asdis', '^1.0.9'),
+        new PackageAndVersion('aoe/captcha-viewhelper', '^1.1.0'),
+        new PackageAndVersion('aoe/dbsequenzer', '^1.0.0'),
+        new PackageAndVersion('aoe/directrequest', '^0.2.0'),
+        new PackageAndVersion('aoe/extbase-filter', '^1.1.1'),
+        new PackageAndVersion('aoe/extbase-functionals', '^0.3.0'),
+        new PackageAndVersion('aoe/extracache', '^0.8.3'),
+        new PackageAndVersion('aoe/feature-flag', '^3.1.7'),
+        new PackageAndVersion('aoe/felogin-bruteforce-protection', '^1.3.7'),
+        new PackageAndVersion('aoe/google-tag-manager', '^0.2.2'),
+        new PackageAndVersion('aoe/happy-feet', '^1.4.3'),
+        new PackageAndVersion('aoe/imgix', '^1.2.2'),
+        new PackageAndVersion('aoe/linkhandler', '^0.0.2'),
+        new PackageAndVersion('aoe/restler', '^0.7.0'),
+        new PackageAndVersion('aoe/sentry-client-js', '^0.4.0'),
+        new PackageAndVersion('aoe/staticpub', '^1.4.0'),
+        new PackageAndVersion('aoe/staticpub-pageexport', '^0.2.0'),
+        new PackageAndVersion('aoe/t3deploy-ext', '^1.4.0'),
+        new PackageAndVersion('aoe/truncate-job', '^0.2.1'),
+        new PackageAndVersion('aoe/update-refindex', '^0.2.5'),
+        new PackageAndVersion('aoe/varnish', '^0.0.1'),
+        new PackageAndVersion('aoepeople/cachemgm', '^2.5.0'),
+        new PackageAndVersion('aoepeople/crawler', '^6.4.1'),
```

Comment:
If I read this correct something is not correct with the parsing. You need ^9.0.0 of the aoepeople/crawler to have it working with TYPO 10.4